lib/test/CMakeLists.txt

```diff
@@ -678,7 +678,9 @@ endif ()
 ot_check_test (ProbabilitySimulationAlgorithm_sensitivity)
 ot_check_test (PostAnalyticalControlledImportanceSampling_std)
 ot_check_test (PostAnalyticalImportanceSampling_std)
+if (Boost_FOUND)
 ot_check_test (Wilks_std)
```

Comment:
If boost is not present, OT should switch to its own algorithms for pBeta. Is it a numerical problem, e.g. a discrepancy in the results, or an algorithmic problem (infinite loop) or something else?

Reply:
yes, it throws in brent

Reply:
I opened an issue for this and removed this commit